cafebabel/articles/tags/models.py

```diff
@@ -39,6 +39,18 @@ def get_images_url(self):
     def get_images_path(self):
         return current_app.config.get('TAGS_IMAGES_PATH')
 
+    def save_from_request(self, request):
```

Comment:
I know we have this for article too and I don't feel so confident about this in the model; it looks more like a view thing.
Let's keep this for now and brain process it for later ;)

Reply:
Agreed, it's very fragile and we have to change it during the next sprint. Maybe using WTForms?

Reply:
[Issue open](https://github.com/cafebabel/cafebabel.com/issues/157)